addons/hitreactions/stringtable.xml

```diff
@@ -3,6 +3,7 @@
     <Package name="hitreactions">
         <Key ID="STR_ACE_hitreactions_minDamageToTrigger_displayName">
             <English>Min Damage to trigger falling</English>
+            <German>Minimaler Schaden um Zusammenbruch auszulösen</German>
```

Comment:
Das hier ist mehr ein "Stolpern" aufgrund der übertragenen Kraft des Auftreffens eines Gefechtskopfes.

Reply:
"Minimaler Schaden zum Auslösen von Stolpern" Might be too long.
"Minimaler Schaden zum Stolpern" Too ambiguous I think?